app/models/pusher.rb

```diff
@@ -97,6 +97,7 @@ def inspect
 
   def after_write
     @version_id = version.id
+    Mailer.delay.gem_pushed(user.id, @version_id) if user
```

Comment:
For simplicity and expediency I opted to skip sending the email here when there's no user. It looks like user is only nil when the rake import task is run.

Reply:
> It looks like user is only nil when the rake import task is run.

Can you please elaborate or provide steps to reproduce what you are seeing?

Reply:
See https://github.com/rubygems/rubygems.org/pull/1950#issuecomment-480515836